packages/api/README.md

```diff
@@ -16,26 +16,68 @@
 
 ## Description
 
-T.B.D.
+This plugin provides an API endpoint for your bot that can be called from external services. A good exemplary use case for this is once your bot grows to have enough configuration options that you want to offer a website to your end-users to change those settings, and your website needs to interface with the bot for this to work.
 
 ## Features
 
 -   Fully ready for TypeScript!
 -   Provides a Decorator-based API to make developing your routes easy!
 -   Follows common REST standards.
 -   Includes ESM ready entrypoint
+-   Premade OAUTH2 endpoint
 
 ## Installation
 
+`@sapphire/plugin-api` depends on the following packages. Be sure to install these along with this package!
+
+-   [`@sapphire/framework`](https://www.npmjs.com/package/@sapphire/framework)
+-   [`@sapphire/pieces v1.x`](https://www.npmjs.com/package/@sapphire/pieces/v/1.2.5)
+-   [`discord.js`](https://www.npmjs.com/package/discord.js)
+-   [`discord-api-types`](https://www.npmjs.com/package/discord-api-types)
+
+You can use the following command to install this package, or replace `npm install` with your package manager of choice.
+
 ```sh
-yarn add @sapphire/plugin-api
+npm install @sapphire/plugin-api @sapphire/framework @sapphire/pieces@1 discord.js discord-api-types
 ```
 
 ---
 
 ## Usage
 
-T.B.D.
+Start by importing the registration file in your project to use the plugin:
+
+**JavaScript**
+
+```js
+require('@sapphire/plugin-api/register');
+```
+
+**TypeScript**
+
+```ts
+import '@sapphire/plugin-api/register';
+```
+
+Then, you can use the following configuration options in your SapphireClient extension class or initializer. This will either be located in your `new SapphireClient` constructor call, or `super` in your constructor method if you use an extension class.
+
+```js
+{
+  auth: {
+    id: '', // The user ID of your Discord bot
+    secret: '', // The client secret of your Discord bot. You can find this at https://discord.com/developers/applications
+    cookie: '', // The name of the authentication cookie. Defaults to "SAPPHIRE_AUTH"
+    redirect: '', // The URL that users should be redirected to after a successful authentication
+    scopes: [], // The scopes that should be given to the authentication. Defaults to ['identify']
+    transformers: '' // Optional transformers to use when you want to transform the raw data from Discord to some data that your website can better understand. Defaults to []
```

Comment:
It takes an array.
```suggestion
    transformers: [] // Optional transformers to use when you want to transform the raw data from Discord to some data that your website can better understand. Defaults to []
```

Reply:
Also, all values should show the plug-in's defaults instead of invalid defaults, and for readability, the comments should be above each field, and limited to 80 characters so there's no wrap nor horizontal scroll.